### ISSUE TITLE
feat: iso action?

### ISSUE THREAD

**Issue by xynydev** [blue-build/github-action#3]
This would be Fedora-specific, obviously. And I'd use the isogenerator action from ublue, it just might be possible to magic it somehow with accepting recipes as inputs or smth.

Would this need it's own repo, though? Anyway, it's not urgent, as we already have a functional ISO action.

**Comment by xynydev** [comment]
This is a necessary feature that should be built soon, once ublue starts using their new isogenerator, we can too.

**Comment by xynydev** [comment]
Apparently the ISOs made by the new isogenerator are too large for GH releases... Supplying some code to upload them to some S3-compatible service would be cool, but I guess we could keep netinstall ISOs around too.

**Comment by gerblesh** [comment]
yeah, it's farily easy to generate the images using the provided docker/podman container locally though

**Comment by xynydev** [comment]
Hmm, yeah. Maybe just a guide then?

**Comment by gmpinder** [comment]
This is going to be available in the CLI at some point. 

Issue: https://github.com/blue-build/cli/issues/151
PR: https://github.com/blue-build/cli/pull/192
